### PR TITLE
Update descriptions for visibility pages and state

### PR DESCRIPTION
**Pull request by froboy**
Previous state:

Cursor_and_Create_Alert___YMCA_of_Central_Ohio

New State:

Cursor_and_Create_Alert___YMCA_of_Central_Ohio

### CODE REVIEW

File: openy_node_alert.install [hunk 1]
```diff
@@ -365,3 +365,26 @@ function openy_node_alert_update_8016() {
...
+    ]
+  ];
+  $config_updater = \Drupal::service('openy_upgrade_tool.param_updater');
+  foreach ($configs as $config_name => $params) {
+    $config = $config_dir . $config_name . '.yml';
+    foreach ($params as $param) {
+      $config_updater->update($config, $config_name, $param);
+    }
+  }
+}
```
Comment (podarok) [inline openy_node_alert.install:390]: fix line, please

File: config/install/field.field.node.alert.field_alert_visibility_state.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ field_name: field_alert_visibility_state
 entity_type: node
 bundle: alert
 label: 'Alert visibility state'
-description: 'Specify pages by using their paths. Enter one path per line. The ''*'' character is a wildcard. An example path is /user/* for every user page. <front> is the front page.'
+description: ''
```
Comment (podarok) [inline config/install/field.field.node.alert.field_alert_visibility_state.yml:19]: why is this removed?
Can we get Screenshot of the form attached to PR?

**Review by podarok — CHANGES_REQUESTED** [review]

File: config/install/field.field.node.alert.field_alert_visibility_state.yml (hunk 2 above)
Comment (froboy) [reply config/install/field.field.node.alert.field_alert_visibility_state.yml:19]: Create_Alert___YMCA_of_Central_Ohio
This description is a duplicate of the one above and does not apply to the `Alert visibility state` field. This field is a simple radio and needs no description.

**Comment by froboy** [comment]
@podarok apologies for the delay. Please re-review.

**Comment by podarok** [comment]
Thanks
Tagged https://github.com/open-y-subprojects/openy_node_alert/releases/tag/1.1.16
